roles/openshift_facts/tasks/main.yml

```diff
@@ -42,8 +42,8 @@
       https_proxy: "{{ openshift_https_proxy | default(None) }}"
       no_proxy: "{{ openshift_no_proxy | default(None) }}"
       generate_no_proxy_hosts: "{{ openshift_generate_no_proxy_hosts | default(True) }}"
-      no_proxy_internal_hostnames: "{{ hostvars | oo_select_keys(groups['oo_nodes_to_config']
-                                                   | union(groups['oo_masters_to_config'])
+      no_proxy_internal_hostnames: "{{ hostvars | oo_select_keys(groups['oo_nodes_to_config'] | default([]) 
+                                                   | union(groups['oo_masters_to_config'] | default([]) ) 
```

Comment:
See https://github.com/openshift/openshift-ansible/issues/1903

oo_*_to_config groups usage should be limited to playbooks only. instead we should be using a separate variable to pass this data into the role.